about.html

```diff
@@ -0,0 +1,80 @@
+    </header>
+
+    <nav aria-label="Main Site Links" class="c-site-nav">
+      <a href="index.html" class="c-site-nav__link">Home</a>
+      <a href="about.html" class="c-site-nav__link">About</a>
+      <a href="projects.html" class="c-site-nav__link">My projects</a>
+      <a href="contact.html" class="c-site-nav__link">Contact</a>
+    </nav>
+
+    <main>
+      <section>
+        <h2 class="c-section__title">About Me</h2>
+        <p class="c-section__text">
+
+    <h2>Background:</h2>
+    <p>
+        With a Master's Degree in Biomedical Science and a rich background as a refugee case worker for over seven years, I bring a versatile skill set to the field of software development.
+        My experience has honed my problem-solving abilities, effective communication, and empathy, making me well-suited for coding challenges, team collaboration, and user-centered design thinking.
+    </p>
+
+    <h2>Tech Skills:</h2>
+    <p>
+        Currently pursuing a career in software development, I am developing my skills in HTML, CSS, SQL, JavaScript, NodeJS, React, and Jest.
+        My commitment to continuous learning allows me to embrace emerging technologies and industry trends.
+    </p>
+
+    <h2>Professional Experience:</h2>
+    <h3>Refugee Case Worker (RETAS LEEDS - Refugee Education Training Advice Service)</h3>
+    <p>Jun 2016 - Present (7 years 3 months)<br>
+        As a refugee case worker, I have supported refugees, asylum seekers, and vulnerable immigrants in rebuilding their lives.
+        My empathy and expertise have empowered them to rebuild with dignity and resilience.
+    </p>
+
+    <h3>Group Facilitator (Ingeus)</h3>
+    <p>Jun 2011 - Jul 2015 (4 years 2 months)<br>
+        In the role of a group facilitator, I led engaging workshops across West Yorkshire, delivering complex information persuasively to diverse audiences.
+    </p>
+
+    <h2>Education:</h2>
+    <p>
+        <strong>Master’s Degree, Biomedical Science, Nottingham Trent University (2015 - 2016)</strong><br>
+        <strong>Postgraduate Diploma (PGDip), Learning and Development Management, The University of Huddersfield (2012 - 2015)</strong>
+    </p>
+
+    <h2>Key Skills:</h2>
+    <ul>
+        <li>Software Development</li>
+        <li>Team Collaboration</li>
+        <li>Problem Solving</li>
+        <li>Continuous Learning</li>
+    </ul>
+
+    <h2>Objective:</h2>
+    <p>Actively seeking software development roles to contribute my expertise to innovative projects and teams.</p>
+
+    <h2>Let's Connect:</h2>
+    <p>Eager to engage with developers, tech enthusiasts, and professionals. Let's connect and explore the endless possibilities in software development!</p>
+
+          <br /><br />
+        </p>
+      </section>
+    </main>
+
+    <footer class="c-site-footer">
+      <p class="c-site-footer__text">&copy; 2023 Fathi. All rights reserved.</p>
+    </footer>
+  </body>
+</html>
```

Comment:
Great job on using semantic HTML @fhkahin ! Your website's structure is well-organised, thanks to the appropriate use of tags like <header>, <nav>, <main>, <section> and <footer>. 
 Additionally, achieving a 100% accessibility score (a11y) demonstrates your commitment to inclusivity. Well done 👏